bin/single_target

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env ruby
+
+require "aws-sdk-elasticloadbalancingv2"
+
+TARGETS_FILE_PATH = '/tmp/deploy_tools_single_tgs'
```

Comment:
As I said - the idea is to save a list of target groups between script run (for example after de-registering to be able to register it back to the same target groups).

Also, even now I can't see anything bad in writing to `/tmp` directory which should exist in pretty-much every Unix-like OS compliant with [FHS](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s18.html) and\or [POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap10.html#tag_10_01). Lots of different tools doing the same and everyone's happy.

Reply:
And also after switching to outputting required command with serializing data I can say - I kinda don't like this way and subjectively would prefer a file-using way.

Even on preprod it creates a pretty big output that you should copy-paste. On prod it will be a lot bigger.

Reply:
okay